modules/virt-about-live-migration.adoc

```diff
@@ -7,14 +7,14 @@
 [id="virt-about-live-migration_{context}"]
 = About live migration
 
-Live migration is the process of moving a running virtual machine instance (VMI) to another node in the cluster without interrupting the virtual workload or access. If a VMI uses the `LiveMigrate` eviction strategy, it automatically migrates when the node that the VMI runs on is placed into maintenance mode. You can also manually start live migration by selecting a VMI to migrate.
+Live migration is the process of moving a running virtual machine instance (VMI) to another node in the cluster without interrupting the virtual workload or access. If a VMI uses the `LiveMigrate` eviction strategy, it automatically migrates when the node that the VMI runs on is placed into maintenance mode.
 
-You can use live migration if the following conditions are met:
+You can manually start live migration by selecting a VMI to migrate.
 
-* Virtual machines must have a persistent volume claim (PVC) with a shared `ReadWriteMany` (RWX) access mode to be live migrated.
+Live migration has the following requirements:
 
-* The pod network binding must not be of the bridge interface type `()`.
+* Shared storage with `ReadWriteMany` (RWX) access mode
 
-* Live migration is supported for virtual machines that are attached to an SR-IOV network interface.
+* Sufficient RAM and network bandwidth
 
-* If the virtual machine uses a host model CPU, live migration is supported only between nodes that support the virtual machine's host model CPU.
+* Appropriate CPUs with sufficient capacity on the worker nodes. If the CPUs have different capacities, live migration might be very slow or fail.
```

Comment:
I think we should keep the note about host model CPUs. It is not obvious (to me) from the new item